x/leveragelp/migrations/v13_migration.go

```diff
@@ -0,0 +1,26 @@
+package migrations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/leveragelp/types"
+)
+
+func (m Migrator) V13Migration(ctx sdk.Context) error {
+	pools := m.keeper.GetAllLegacyPools(ctx)
+
+	for _, pool := range pools {
+		new_pool := types.Pool{
+			AmmPoolId: pool.AmmPoolId,
+			Health: pool.Health,
+			Enabled: pool.Enabled,
+			Closed: pool.Closed,
+			LeveragedLpAmount: pool.LeveragedLpAmount,
+			LeverageMax: pool.LeverageMax,
+			MaxLeveragelpPercent: sdk.NewDec(60),
```

Comment:
@cryptokage1996 why do we have both `LeverageMax` and `MaxLeveragelpPercent` in the store aren’t those two things the same data?

Reply:
@cryptokage1996 got it thanks for clarifying although do we need to store the value of `MaxLeveragelpPercent` can we instead calculate it on the fly when it is queried?

Reply:
@cryptokage1996 ok that makes more sense it acts as a threshold, sounds good